Examples/ExampleServer/Program.cs

```diff
@@ -29,6 +29,8 @@ static bool AcceptIncomingStream(IStream stream)
     static byte[] responseBody = Encoding.ASCII.GetBytes(
         "<html><head>Hello World</head><body>Content</body></html>");
 
+    private static System.Net.Security.SslServerAuthetnicationOptions options;
```

Comment:
I think this change here is not needed, since the SSL changes got into a different example?